config/schema/entity_meta.entity_type.schema.yml

```diff
@@ -0,0 +1,10 @@
+emr.entity_meta_type.*:
```

Comment:
You can put all the schemas in one schema file: `emr.schema.yml` This file is also named incorrectly.